docs/interfaces/rtc.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Real Time Clock (internal)
```

Comment:
move this to mx4  rtc under linux 3.1 
and create an new file rtc with timedatectl(https://gitlab.com/hostmobility/mx5-tools/-/blob/master/user_documentation/realtime_clock.md?ref_type=heads).

add chapter how to wakeup on the rtc. set alarm (for mx4 most user use go_to_sleep.sh)